src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -187,6 +187,23 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
         return filters;
     }
 
+    /**
+     * Get the list of default plugins used for this instance of this descriptor. Used for help/doc generation.
+     * @return List of T
+     */
+    @Override
+    public List<Object> getDefaultInstances() { return new ArrayList<>(toolDefaultReadFilters.values()); }
```

Comment:
I think that what you say is quite reasonable, @cmnbroad. I will wait til the decision is made.